api/src/main/java/com/velocitypowered/api/command/Command.java

```diff
@@ -14,9 +14,10 @@
    * Executes the command for the specified {@link CommandSource}.
    *
    * @param source the source of this command
+   * @param alias the alias used to execute this command
    * @param args the arguments for this command
    */
-  void execute(CommandSource source, String @NonNull [] args);
+  void execute(CommandSource source, String alias, String @NonNull [] args);
```

Comment:
This change breaks _every_ existing Velocity plugin that has a command. This is unacceptable.

Reply:
Would that be possible in the re-write of the command system in the near future?

Reply:
There would be a compatibility shim that would be compatible with the current command API, and then dropped in a major release (TBD).